ds3_java_cli/src/main/java/com/spectralogic/ds3cli/util/Utils.java

```diff
@@ -38,7 +38,7 @@ public FileVisitResult visitFile(final Path file, final BasicFileAttributes attr
         return objectsBuilder.build();
     }
 
-    public static String getFileName(final Path rootDir, final Path filePath) {
+    public static String getFileName(final Path rootDir, final Path filePath) throws IOException{
```

Comment:
Was the `throws IOException` required?  If this worked before, I'd rather we not include it.